js/common/lib/session-impl-utils.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {InferenceSession as InferenceSession} from './inference-session.js';
+
+export const processModel = (model: string|ArrayBufferLike|Uint8Array): string|Uint8Array => {
+  if (typeof model === 'string' || model instanceof Uint8Array) {
+    return model;
+  } else if (model instanceof ArrayBuffer) {
+    return new Uint8Array(model);
+  } else {
+    throw new TypeError('Unexpected argument: must be \'path\' or \'buffer\'.');
+  }
+};
+
+export const processModelOrOptions =
```

Comment:
with the new signature of `TrainingSession.create`, you probably don't need this anymore.